ADR/World/Constants/Constants.py

```diff
@@ -1,3 +1,2 @@
-class Constants():
-    def __init__(self):
-        self.gravity = 9.831
\ No newline at end of file
+def get_gravity_constant():
+    return 9.7925
```

Comment:
Please, [add a new line at the end of the file](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file).